addons/l10n_cl/hooks.py

```diff
@@ -0,0 +1,21 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.api import Environment, SUPERUSER_ID
+import logging
+
+
+def update_tax_calculation_rounding_method(cr, registry):
+    env = Environment(cr, SUPERUSER_ID, {})
+    env['res.company'].search(
+        [('partner_id.country_id', '=', env.ref('base.cl').id)]).write({
+            'tax_calculation_rounding_method': 'round_globally', })
```

Comment:
AR did this differently.  (by inheriting the generate chart template) to remove the post_init_hook.

Reply:
Removed in f4b9c22edc0